src/Sarif.Sdk.sln

```diff
@@ -96,6 +96,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		.editorconfig = .editorconfig
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ConsoleApp1", "ConsoleApp1\ConsoleApp1.csproj", "{416A443A-B0C0-44A8-AA4A-B1221B79F6C2}"
```

Comment:
>ConsoleApp1 [](start = 80, length = 11)

Is this a new project?

Reply:
Test project I used to produce a title from every checked in SARIF result. :) Updated the SLN to remove this reference.

---
In reply to: [392436877](https://github.com/microsoft/sarif-sdk/pull/1813#discussion_r392436877) [](ancestors = 392436877)